src/processors/models/database/cassandra/cassandraConnector.js

```diff
@@ -7,13 +7,15 @@ const INSERT_PROCESSOR = 'INSERT INTO processors(id, name, description, type, fi
 const GET_ALL_PROCESSORS = 'SELECT * FROM processors';
 const GET_PROCESSOR = 'SELECT * FROM processors where id=?';
 const DELETE_PROCESSOR = 'DELETE FROM processors WHERE id=?';
+const UPDATE_PROCESSOR = 'UPDATE processors SET name=?, description=?, type=?, file_url=?, javascript=?, updated_at=?  WHERE id=? AND created_at=? IF EXISTS';
```

Comment:
we shouldn't need `AND created_at=?` in this update query

Reply:
if I remove it, Cassandra will ail updating, claiming created_at is missing